tests/jax/test_fused_attn.py

```diff
@@ -305,11 +305,14 @@ def _check_configs(self):
         ]:
             pytest.skip("THD format requires padding masks.")
 
-        if self.qkv_layout == QKVLayout.BS3HD or get_qkv_format(self.qkv_layout) == QKVFormat.THD:
-            if self.num_heads_q != self.num_heads_kv:
-                pytest.skip("QKVPACKED layout requires num_heads_q and num_heads_kv to be equal.")
+        qkv_format = get_qkv_format(self.qkv_layout)
+        if self.qkv_layout == QKVLayout.BS3HD or qkv_format == QKVFormat.THD:
             if self.max_seqlen_q != self.max_seqlen_kv:
-                pytest.skip("QKVPACKED layout requires max_seqlen_q and max_seqlen_kv to be equal.")
+                pytest.skip(f"{self.qkv_layout} requires max_seqlen_q == max_seqlen_kv")
```

Comment:
Why do we skip cross attention here for THD? It should work for THD_T2HD and THD_TH2D cases as well right?

Reply:
Nice catch, PMed you.

Reply:
Thanks for the details. We'll merge the PR and wait for the cross-attention related bug to be fixed then. Thanks.